repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -269,15 +269,25 @@ def prepare_target_userspace(context, userspace_dir, enabled_repos, packages):
             # failed since leapp does not support updates behind proxy yet.
             for manager_info in api.consume(PkgManagerInfo):
                 if manager_info.configured_proxies:
-                    details['details'] = ("DNF failed to install userspace packages, likely due to the proxy "
-                                          "configuration detected in the YUM/DNF configuration file.")
+                    details['details'] = (
+                        "DNF failed to install userspace packages, likely due to the proxy "
+                        "configuration detected in the YUM/DNF configuration file. "
+                        "Make sure the proxy is properly configured in /etc/dnf/dnf.conf. "
+                        "It's also possible the proxy settings in the DNF configuration file are "
+                        "incompatible with the target system. A compatible configuration can be "
+                        "placed in /etc/leapp/files/dnf.conf which, if present, will be used during "
+                        "the upgrade instead of /etc/dnf/dnf.conf. "
+                        "In such case the configuration will also be applied to the target system."
```

Comment:
I missed to update this msg. possibly we will update it as a follow up. the text of the msg has no effect on the functionality, so I am in favor to merge it now to allow the testing. The problem I see with this msg is that people could thing they need to configure correctly just /etc/leapp/files/dnf.conf file, but the truth is that /etc/leapp/files needs to be still configured properly on the src system as well.